src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -855,26 +857,27 @@ def log_truncated_removed_packages_if_any(self):
         if not len(self.__installation_packages_removed) == 0:
             self.composite_logger.log_debug("Installation truncated removed packages : {0}".format(self.__installation_packages_removed))
 
+    # rename __calc_status_size_on_disk, __calc_package_payload_size_on_disk
     def __write_status_file(self, complete_status_file_payload):
         """ Truncate the substatus summary patch list when complete status file size is more than 126kb """
         """
             __write_status_file(self, complete_status_file_payload):
-                status_file_size = __get_byte_size(complete_status_file_payload)
+                status_file_size = __calc_status_size_on_disk(complete_status_file_payload)
                 if status_file_size > 126kb
                     _index, _name = self.__get_index_name()
-                    empty_list_file_size = __get_twice_byte_size(__size_of_constant_status_data(complete_status_file_payload))
-                    packages_size_limit = 126kb - empty_list_file_size
+                    status_file_no_list_size = __calc_package_payload_size_on_disk(__get_size_of_constant_status_data(complete_status_file_payload))
```

Comment:
rename status_file_no_list_size to status_file_without_package_list_size or status_file_size_minus_package_list_size